packages/analyticsdx-template-lint/src/constants.ts

```diff
@@ -35,6 +35,7 @@ const definitionFilesPats = [
   ['variableDefinition'] as JSONPath,
   ['uiDefinition'],
   ['layoutDefinition'],
+  ['readinessDefinition'],
```

Comment:
This makes some things (like CMD+Click) work against `readinessDefinition` in template-info.json